microshift_support/microshift-sos-report.adoc

```diff
@@ -7,7 +7,7 @@ include::_attributes/attributes-microshift.adoc[]
 toc::[]
 
 [role="_abstract"]
-`sos` is a tool you can use to collect troubleshooting information about a host. An `sos report` will generate a detailed report with all the enabled plugins and data from the different components and applications in a system.
+`sos` is a tool you can use to collect troubleshooting information about a host. An `sos report` command generates a detailed report showing all of the enabled plugins and data from the different components and applications in a system.
```

Comment:
Per [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=grammar-capitalization#general-text):

> Do not start a sentence with a lowercase term; rewrite the sentence.

For example:
```suggestion
You can use the`sos` tool to collect troubleshooting information about a host. An `sos report` command generates a detailed report showing all of the enabled plugins and data from the different components and applications in a system.
```